### ISSUE TITLE
"Transfer failed" error on import from UI

### ISSUE THREAD

**Issue by radinamatic** [learningequality/kolibri#2495]
### Observed behavior

Observed on several instances of the 0.6 release ([demo server](http://le-060-beta.learningequality.org), see discussion [in another issue](https://github.com/learningequality/kolibri/issues/2419#issuecomment-335950339)), and dev builds ([most recently here](https://github.com/learningequality/kolibri/pull/2494#issuecomment-338353686)): importing more than one channel fails with this unhelpful message. Import from the command line works fine.

![manage device content - kolibri - google chrome_688](https://user-images.githubusercontent.com/1457929/31865843-e238fbc8-b775-11e7-8ea0-051590eeae95.png)

### Expected behavior

Being able to import more than one channel.

### Errors and logs

None :unamused: 

### Steps to reproduce

Try to import more than one content channel from the UI.

### Context

* Kolibri version: release 0.6
* Operating system: all
* Browser: Chrome & FF

cc @rtibbles

**Comment by indirectlylit** [comment]
cc @aronasorman @jonboiser any thoughts?

**Comment by aronasorman** [comment]
@radinamatic did you see any traceback on the CLI when the error occurred?

**Comment by benjaoming** [comment]
Additional info: This has been uncaught by tests, we should reproduce it in a test..

**Comment by radinamatic** [comment]
> any traceback on the CLI when the error occurred?

None, zilch, nada, cap, nista... :confused:

**Comment by rtibbles** [comment]
My guess is something is happening in the frontend to misread the task?

**Comment by rtibbles** [comment]
Can you paste the 'preview' when clicking on the tasks request in the network tab?

**Comment by radinamatic** [comment]
Hmmm, I can't seem to reproduce the error on the PEX from the [same build as in this comment](https://github.com/learningequality/kolibri/pull/2494#issuecomment-338353686). 
Will fire up a Windows VM to retry. 

Ah, and btw @rtibbles, admin in this build is being logged out, and not even after 10 minutes... :confused:

**Comment by rtibbles** [comment]
Yeah, the timeout fixes have not propagated to 0.6 yet, they are coming.

**Comment by benjaoming** [comment]
I can reproduce this, happens when important the second channel, no useful log output:

```
INFO     Downloading data for channel id 8b4d3e6d3d4842ba8ea658335b5dd252
INFO:kolibri.content.management.commands.importchannel:Downloading data for channel id 8b4d3e6d3d4842ba8ea658335b5dd252
^M  0%|          | 0/154624 [00:00<?, ?it/s]^M100%|##########| 154624/154624 [00:00<00:00, 801173.16it/s]INFO     Importing ContentTag data
INFO:kolibri.content.utils.channel_import:Importing ContentTag data
INFO     Importing ContentNode_has_prerequisite data
INFO:kolibri.content.utils.channel_import:Importing ContentNode_has_prerequisite data
INFO     Importing ContentNode_related data
INFO:kolibri.content.utils.channel_import:Importing ContentNode_related data
INFO     Importing ContentNode_tags data
INFO:kolibri.content.utils.channel_import:Importing ContentNode_tags data
```

**Comment by rtibbles** [comment]
Has the transfer actually failed, or is the frontend just saying that it has failed?

**Comment by benjaoming** [comment]
Yes, it failed, my login also timed out, got distracted by another potential issue, will return with more :)

**Comment by rtibbles** [comment]
Yeah, any traceback would be good!

**Comment by rtibbles** [comment]
Login timeout should now be fixed on 0.6

**Comment by benjaoming** [comment]
Possible scenario:

1. Download Channel A (takes a while)
1. Finishes
1. 4½ minutes pass, I come back and enter the second channel
1. Login times out while import process has just launched

Also, I can be logged out while importing the second channel, this could be because login session isn't kept alive as it should after importing the first channel?

![image](https://user-images.githubusercontent.com/374612/31951116-8384f712-b8dd-11e7-8e58-b65521558aad.png)

**Comment by aronasorman** [comment]
Here's what I tried:
1. Created a new Vagrant VM running Ubuntu Zesty64
1. Created a new Kolibri pex file from the latest u/release-0.6.1. Commit: `c1378bb664ca7f949ba71041a5b32f668fd3437d`
1. Started said pex file from inside the Vagrant VM
1. Created a self-manged installation, and an admin account
1. Imported `8b4d3e6d3d4842ba8ea658335b5dd252` first. Success.
1. Imported `8b4d3e6d3d4842ba8ea658335b5dd252` again. Success.
1. Import portion control: `fb51dae6df7545af8455aa3a0c32048d`. Success.
1. Import a small test channel: `4d8725009c1d4c6fa81f61d5c2a51f3d`. Success
1. Import one of the MagoGenie channels: `8e06f21efdf23ffab4b026f674a8c442`. Success

Maybe @radinamatic or @benjaoming either of you can join me in a screenshare session where we can try to replicate this live?

**Comment by radinamatic** [comment]
@benjaoming Is a better bet, since I've killed the VM where the issue appeared last... :disappointed: 
Or @benjaoming can tell me which installer he used, I can grab it for the session with @aronasorman...?

**Comment by benjaoming** [comment]
@aronasorman try the exact same PEX as @radinamatic linked ( https://github.com/learningequality/kolibri/pull/2494 ) -- since we are looking for something that has been confirmed several times, but has not been consistently reproduced, we have to have as few unknowns as possible :/

I can say this much: I encountered it when importing `bcd99d8aeef04ce6b9e25a88d87eedb7` (demo) followed by `8b4d3e6d3d4842ba8ea658335b5dd252` (KA test)

**Comment by benjaoming** [comment]
Will try to reproduce and get back!

**Comment by radinamatic** [comment]
Weird, because I did try to repro it with that same PEX 2 days ago, to no avail... I then tried with Windows installer from the same build, no luck either. I even put in the sweet time to import manually the full CK-12 channel (6GB) and then try some re-import over it, still nothing... :rage: 

Not sure what is more annoying, seeing the bug all the time when it's messing your workflow, or not being able to repro it when you actually want to debug it :roll_eyes:

**Comment by rtibbles** [comment]
Fixed in #2564
